.github/ISSUE_TEMPLATE/documentation-bug.yaml

```diff
@@ -0,0 +1,26 @@
+name: 🐛 Documentation Bug
+description: Report a bug that affects Material Web documentation (https://material-web.dev/)
+
+body:
+  - type: textarea
+    id: description
+    validations:
+      required: true
+    attributes:
+      label: Description
+      description: |
+        Please describe the documentation issue in as much detail as possible.
+      placeholder: |
+        Add issue description here.
+
+  - type: textarea
+    id: environment
+    validations:
+      required: true
```

Comment:
Prolly optional if it's not a catalog bug

Reply:
Done!